web-animations-2/Overview.bs

```diff
@@ -1014,6 +1015,7 @@ Add:
 >     matching condition from below:
 >
 > <div class="switch">
+>
```

Comment:
Same here - indent the markdown list and you won't need these blank lines to hack around the markdown parser. And swap the `div` for a `dl` as well.

Reply:
I've swapped all of them now ^_^